packages/nimble-components/package.json

```diff
@@ -50,7 +50,9 @@
     "@microsoft/fast-foundation": "^2.46.11",
     "@microsoft/fast-web-utilities": "^5.4.1",
     "@ni/nimble-tokens": "^4.3.0",
+    "@types/d3-color": "^3.1.0",
```

Comment:
We should be able to list `@types/` dependencies under `devDependencies` instead of `dependencies`. They're only used at build time for this project, not shipped as part of the bundle or installed in applications. You can move `@types/d3-scale` too (I should have caught this when it was first added, oops). Don't forget to re-run `npm install` after doing this to regenerate `package-lock.json`.

Reply:
[Here](https://github.com/ni/nimble/pull/831#discussion_r1029733907) Milan said it's fine where they are? Did something change from then?

Reply:
Yea I *think* that it's useful to keep as dependencies so that applications that build with [`skipLibCheck: false`](https://www.typescriptlang.org/tsconfig#skipLibCheck) are able to build against a reliable set of types.

But I also haven't seen that done elsewhere so I'm not sure if it's a common practice. I think I lean toward keeping them until it's an issue.

Separately, when talking to @munteannatan we discussed that if we are just using d3-color for simple color conversions then maybe we can use https://www.npmjs.com/package/@microsoft/fast-colors instead. Eventually we should switch our hex-rgb usage to that as well so we aren't pulling in multiple librraries for the same responsibility (hex-rgb wasn't useful for this specific use case).

If you end up switching to fast-colors I'd like to take a look and you can reset me.

Reply:
Seems to be an area that the Nimble team should think through in more detail sometime so that we can provide clear guidelines. But that doesn’t have to be now; feel free to leave it as-is for this PR.